src/main/java/com/github/manolo8/darkbot/utils/Base64Utils.java

```diff
@@ -1,11 +1,25 @@
 package com.github.manolo8.darkbot.utils;
 
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
 import java.util.Base64;
 
 public class Base64Utils{
 
-    public static String base64Decode(String text) throws Exception {
-        return new String(Base64.getDecoder().decode(text),"UTF-8");
+    public static String base64Decode(InputStream input) throws Exception {
```

Comment:
I didn't expect you to remove the string method, just to add the input stream one aswell